Source/WebKit/DerivedSources-input.xcfilelist

```diff
@@ -29,6 +29,7 @@ $(JAVASCRIPTCORE_PRIVATE_HEADERS_DIR)/generator_templates.py
 $(JAVASCRIPTCORE_PRIVATE_HEADERS_DIR)/jsmin.py
 $(JAVASCRIPTCORE_PRIVATE_HEADERS_DIR)/models.py
 $(JAVASCRIPTCORE_PRIVATE_HEADERS_DIR)/xxd.pl
+$(PROJECT_DIR)/$(WK_MODULEMAP_PRIVATE_FILE)
```

Comment:
Does this work? I would have thought that the generator would insist on having separate `$(PROJECT_DIR)/Modules/…` paths, because it only does a few hard-coded variable substitutions.

Reply:
Yeah, I just ran a build locally and got:
```diff
diff --git a/Source/WebKit/DerivedSources-input.xcfilelist b/Source/WebKit/DerivedSources-input.xcfilelist
index 6e57f3b3df81..c9a80ed71b64 100644
--- a/Source/WebKit/DerivedSources-input.xcfilelist
+++ b/Source/WebKit/DerivedSources-input.xcfilelist
@@ -98,6 +98,7 @@ $(PROJECT_DIR)/GPUProcess/webrtc/RemoteMediaRecorder.messages.in
 $(PROJECT_DIR)/GPUProcess/webrtc/RemoteMediaRecorderManager.messages.in
 $(PROJECT_DIR)/GPUProcess/webrtc/RemoteSampleBufferDisplayLayer.messages.in
 $(PROJECT_DIR)/GPUProcess/webrtc/RemoteSampleBufferDisplayLayerManager.messages.in
+$(PROJECT_DIR)/Modules/OSX_Private.modulemap
 $(PROJECT_DIR)/NetworkProcess/Cookies/WebCookieManager.messages.in
 $(PROJECT_DIR)/NetworkProcess/CustomProtocols/LegacyCustomProtocolManager.messages.in
 $(PROJECT_DIR)/NetworkProcess/NetworkBroadcastChannelRegistry.messages.in
```

We'll need the real paths :\

Reply:
It worked when I had it in alphabetical order as if WK_MODULEMAP_PRIVATE_FILE was expanded, let me try moving it back.

Reply:
Can't seem to get it to work now 🤔 Oh well, individual paths it is.

Reply:
As an aside, why is Derived Sources set up as an aggregate target with a Run Script build phase instead of an External Build System target? It'd save some headache of maintaining an xcfilelist I think?